spec/system/task_spec.rb

```diff
@@ -1,11 +1,8 @@
-# frozen_string_literal: true
-
 require 'rails_helper'
 RSpec.describe 'タスク管理機能', type: :system do
-
   describe 'タスク一覧画面' do
     before do
-      @task = FactoryBot.create(:task, task_name: 'task')
+      @task = FactoryBot.create(:task)
```

Comment:
設定ファイルに追記することで`FactoryBot.`は省略することができます。
```suggestion
      @task = create(:task)
```